<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PTorusMonitor</class>
 <widget class="QWidget" name="PTorusMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Torus Monitor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabs_Main">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_General">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="grp_Stats">
           <property name="title">
            <string>Runtime Statistics</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::ExpandingFieldsGrow</enum>
              </property>
              <property name="labelAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="lbl_RunningTime">
                <property name="text">
                 <string>Running Time: 00:00:00</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lbl_noIterations">
                <property name="text">
                 <string>Lucy Iterations: 0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="grid_GridData">
           <property name="title">
            <string>Grid Information</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <property name="labelAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="lbl_geometryType">
                <property name="text">
                 <string>Geometry: </string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lbl_maxDepth">
                <property name="text">
                 <string>Max AMR Depth</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="lbl_halfSmallestSubcell">
                <property name="text">
                 <string>Half Smallest Subcell: </string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="lbl_noOctals">
                <property name="text">
                 <string>No. of Octals</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="lbl_noVoxels">
                <property name="text">
                 <string>No. of Voxels: </string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="lbl_smoothingFactor">
                <property name="text">
                 <string>Smoothing Factor: </string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="lbl_gridCentre">
                <property name="text">
                 <string>Grid Centre: </string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="lbl_sizeLargestCell">
                <property name="text">
                 <string>Size of Largest Cell: </string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Performance">
      <attribute name="title">
       <string>Performance</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QTableWidget" name="tbl_eventTable">
         <property name="columnCount">
          <number>3</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>190</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Event</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Event Description</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Time</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QwtPlot" name="plt_TimeHist">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>75</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Convergence">
      <attribute name="title">
       <string>Convergence</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QwtPlot" name="grp_convPlot"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
